src/main/java/teammates/ui/controller/Action.java

```diff
@@ -452,12 +453,39 @@ public String getRequestParamValue(String paramName) {
     }
     
     /**
+     * Retrieves the value for the specified parameter expected to be present in the http request.
+     *
+     * @param paramName  a constant from the {@link Const.ParamsNames} class.
+     * @return the value of the specified parameter if it exists, asserts otherwise.
+     */
+    public String getExpectedRequestParamValue(String paramName) {
+        String value = HttpRequestHelper.getValueFromParamMap(requestParameters, paramName);
+        Assumption.assertPostParamNotNull(paramName, value);
+        return value;
+    }
+    
+    /**
      * @return null if the specified parameter was not found in the request.
      */
     public String[] getRequestParamValues(String paramName) {
         return HttpRequestHelper.getValuesFromParamMap(requestParameters, paramName);
     }
     
+    /**
+     * Retrieves the values for the specified parameter expected to be present in the http request.
+     *
+     * @param paramName  a constant from the {@link Const.ParamsNames} class.
+     * @return the values of the specified parameter if it exists, asserts otherwise.
+     */
+    public String[] getExpectedRequestParamValues(String paramName) {
+        String[] values = HttpRequestHelper.getValuesFromParamMap(requestParameters, paramName);
+        if (values == null) {
+            throw new NullPostParameterException(String.format(Const.StatusCodes.NULL_POST_PARAMETER,
+                    paramName));
+        }
+        return values;
+    }
+    
```

Comment:
Same here, reuse the original method.

Also, perhaps overloading the `assertPostParamNotNull` method in Assumption.java to take in a `String[]`as the second parameter (i.e. `assertPostParamNotNull(String parameterName, String[] postParameterArray)`) may be helpful, since then you can just use that method instead of re-implementing an almost identical version here.

Actually, it may be better if `assertPostParamNotNull` was made generic i.e.
```
public static <T> void assertPostParamNotNull(String parameterName, T postParameter) {
    if (postParameter == null) {
        throw new NullPostParameterException(String.format(Const.StatusCodes.NULL_POST_PARAMETER, parameterName));
    }
}
```
, since all it is doing is checking if the value is `null`, and the null value is of every type. But there may be reasons for not doing that, so I'll let the senior reviewers comment on that.